tensorrt_llm/llmapi/llm.py

```
@@ -198,13 +198,11 @@ def __init__(self,
             if not self.mpi_session:
                 mpi_process_pre_spawned: bool = get_spawn_proxy_process_env()
                 if not mpi_process_pre_spawned:
-                    print_colored_debug(f"LLM create MpiPoolSession\n",
-                                        "yellow")
+                    logger_debug(f"LLM create MpiPoolSession\n", "yellow")
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)